src/tab/tab-link-chain.tsx

```diff
@@ -13,18 +13,24 @@ import { ResizeCallbackParams } from "../shared/fade-wrapper";
 import { Breakpoint } from "../theme";
 import { useTheme } from "styled-components";
 
-interface Props {
-    controlledMode?: boolean | undefined;
+export interface TabLinkChainBaseProps {
     "data-testid"?: string | undefined;
     onTabClick?: ((title: string, order: number) => void) | undefined;
     fullWidthIndicatorLine?: boolean | undefined;
+    overrideTabWidth?: string | undefined;
+    centerTabLabel?: boolean | undefined;
```

Comment:
the tab label could be centered by default anyway? the current behaviour "hugs" the tab label so it doesn't make a visual difference